app/src/main/java/org/schabi/newpipe/info_list/holder/StreamMiniInfoItemHolder.java

```diff
@@ -81,6 +86,10 @@ public void updateFromItem(final InfoItem infoItem,
             itemProgressView.setVisibility(View.GONE);
         }
 
+        if (!DependentPreferenceHelper.getPositionsInListsEnabled(itemProgressView.getContext())) {
+            itemProgressView.setVisibility(View.GONE);
+        }
+
```

Comment:
I don't think you need these lines. If you look above, every branch of the `if`s sets the `itemProgressView`'s visibility.